src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+

```

Comment:
remove empty line

Reply:
Thank you for your request, but Maven can't build code without these empty lines(

Reply:
not fixed